corehq/motech/fhir/views.py

```diff
@@ -84,15 +84,15 @@ def search_view(request, domain, fhir_version_name, resource_type):
     fhir_version = _get_fhir_version(fhir_version_name)
     if not fhir_version:
         return JsonResponse(status=400, data={'message': "Unsupported FHIR version"})
-    patient_case_id = request.GET.get('patient_id')
-    if not patient_case_id:
-        return JsonResponse(status=400, data={'message': "Please pass patient_id"})
+    resource_id = request.GET.get('_id')
```

Comment:
I'm a bit unclear about the implications of changing `patient_id` to `_id` here, since we expect this parameter now. Are there other places that should also be updated (excpet the tests that are) in order to accomodate this change?

Reply:
@AddisonDunn the issue is that we're changing a GET parameter name so any callers of the API will also need to be changed.

(looks like I didn't submit my last set of feedback)

Reply:
I think the easiest thing for now is to keep `patient_id` as 2nd option. We can investigate removing it later.